app/views/layouts/application.html.erb

```diff
@@ -89,7 +89,8 @@
                   <%= flash[:alert] %>
                 </div>
               <% end %>
-            <%= yield %>
+              <%= yield %>
+            </div>
```

Comment:
oops! good catch!

Reply:
Yeah, I was doing some bits in the code and it kept trying to add a damn div and finally worked out why